source/_integrations/telekomunikace_2n.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: 2N Telekomunikace
+description: Instructions on how to set up 2N Telekomunikace in Home Assistant.
+ha_category:
+  - Button
+  - Sensor
+  - Switch
```

Comment:
Oh. I wasn't aware of that rule, but it makes sense, in order to decrease review work. I'll keep the button platform, because the functions controlled by these are supported by the most devices.